src/main/java/bio/terra/cloudfunctions/common/GsonWrapper.java

```diff
@@ -0,0 +1,43 @@
+package bio.terra.cloudfunctions.common;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonDeserializer;
+import java.time.OffsetDateTime;
+import java.time.ZonedDateTime;
+
+public class GsonWrapper {
+  private static Gson _instance = initInstance();
+  /**
+   * Returns a Gson instance capable of parsing a string that represents OffsetDateTime object.
+   *
+   * @return Gson object
+   */
+  private static Gson initInstance() {
+    if (_instance == null)
+      _instance =
+          new GsonBuilder()
+              .setFieldNamingPolicy(FieldNamingPolicy.IDENTITY)
+              .registerTypeAdapter(
```

Comment:
See PR#11 for thread-safe version (still in review).

Otherwise, everything looks good!